### ISSUE TITLE
RadioButton does not render the circle/selection in UWP

### ISSUE THREAD

**Issue by vchelaru** [enisn/Xamarin.Forms.InputKit#140]
**Describe the bug**
Radio buttons (which render properly on iOS and Android) do not render properly on UWP.

**To Reproduce**
1. Create a UWP project
2. Create a page
3. Add radio buttons to the page
4. Run the app in UWP
5. Observe that radio buttons do not show their left-side graphics, but text does render

Additional information: The radio buttons do function properly - they can be checked and they do update the view model properly, so it seems to be a visual issue.

** Example Code **

```
var group = new RadioButtonGroupView();
stackLayout.Children.Add(group);

var firstRadio = new RadioButton();
firstRadio.TextColor = Color.Black;
firstRadio.Text = "First";
group.Children.Add(firstRadio);


var secondRadio = new RadioButton();
secondRadio.TextColor = Color.Black;
secondRadio.Text = "Second";
group.Children.Add(secondRadio);

var thirdRadioButton = new RadioButton();
thirdRadioButton.TextColor = Color.Black;
thirdRadioButton.Text = "Third";
group.Children.Add(thirdRadioButton);

var fourthRadioButton = new RadioButton();
fourthRadioButton.TextColor = Color.Black;
fourthRadioButton.Text = "Fourth";
group.Children.Add(fourthRadioButton);
```

**Expected behavior**
The radio button should render its circle, and the circle should be filled-in if the radio button is checked.

**Screenshots**
![image](https://user-images.githubusercontent.com/729631/58586063-7a58a800-8217-11e9-99d2-6a2bd768378a.png)

**Desktop (please complete the following information):**
 - OS: Windows 10

**Smartphone (please complete the following information):**
 - Device: Motorola Moto G6
 - OS: Android 8.0

**Additional context**

**Comment by cortensinger** [comment]
Just want to say that I am experiencing the same bug with UWP. I know that support for UWP is officially "EXPERIMENTAL" but would love to see this fixed :) thanks

**Comment by enisn** [comment]
UWP is not offically supported. I just merge new Pull Request for UWP.

**Comment by Nambukarthy** [comment]
Any workaround for this ?

**Comment by AhmedAdelGadElkareeem** [comment]
still the same problem @enisn

**Comment by enisn** [comment]
https://github.com/enisn/Xamarin.Forms.InputKit/blob/rel-3.7/InputKit/Platforms/UWP/IconViewRenderer.cs

IconView implementation doesn't work properly on UWP. It should create an image and tint with given color but it can't do that after a version of UWP but I'm not sure which one it is.

**Comment by enisn** [comment]
This issue is closed since v4.0 supports Windows platform
